Debounce/Generic/DebouncedEventArgs.cs

```diff
@@ -7,60 +7,57 @@
 namespace Dorssel.Utilities.Generic;
 
 /// <summary>
-/// Provides data for the <see cref="Debouncer{TData}.Debounced"/> event.
+/// Provides data for the <see cref="IDebouncerBase{TEventArgs}.Debounced"/> event.
 /// </summary>
-public class DebouncedEventArgs<TData> : EventArgs
+public class DebouncedEventArgs<TData> : DebouncedEventArgs
 {
     /// <summary>
     /// Initializes a new instance of the <see cref="DebouncedEventArgs{TData}"/> class.
     /// </summary>
-    /// <param name="count">The number of triggers accumulated since the previous event was sent.
-    /// <para>Must be greater than 0.</para>
-    /// </param>
+    /// <param name="count"></param>
     /// <param name="triggerData">
-    /// Accumulated data from each individual trigger, or empty when buffering is disabled in the <see cref="Debouncer{TData}"/>
+    /// Accumulated data from each individual trigger. Must not be empty."/>
     /// </param>
-    /// <exception cref="ArgumentOutOfRangeException">Thrown when <paramref name="count"/> is not greater than 0.</exception>
+    /// <exception cref="ArgumentOutOfRangeException">Thrown when <paramref name="triggerData"/> is an empty list.</exception>
```

Comment:
Documentation does not match the actual check, which seems to check whether `triggerData` contains more items than `count`, not the empty list check.

Reply:
Fixed